README.md

```diff
@@ -4,18 +4,24 @@
 
 # pgai
 
-<h3>pgai brings AI workflows to your PostgreSQL database</h3>
+<h3>pgai allows you to develop RAG, semantic search, and other AI applications directly in PostgreSQL</h3>
 
 [![Discord](https://img.shields.io/badge/Join_us_on_Discord-black?style=for-the-badge&logo=discord&logoColor=white)](https://discord.gg/KRdHVXAmkp)
 [![Try Timescale for free](https://img.shields.io/badge/Try_Timescale_for_free-black?style=for-the-badge&logo=timescale&logoColor=white)](https://tsdb.co/gh-pgai-signup)
 </div>
 
-pgai simplifies the process of building [search](https://en.wikipedia.org/wiki/Similarity_search), and
-[Retrieval Augmented Generation](https://en.wikipedia.org/wiki/Prompt_engineering#Retrieval-augmented_generation) (RAG) AI applications with PostgreSQL.
+pgai simplifies the process of building [search](https://en.wikipedia.org/wiki/Similarity_search),
+[Retrieval Augmented Generation](https://en.wikipedia.org/wiki/Prompt_engineering#Retrieval-augmented_generation) (RAG), and other AI applications with PostgreSQL.
 
-pgai brings embedding and generation AI models closer to the database. With pgai, you can now do the following directly from within PostgreSQL in a SQL query:
+The goal of pgai is to make working with AI easier and more accessible to developers. Because data is
+the foundation of most AI applications, pgai makes it easier to leverage your data in AI workflows. In particular, pgai supports:
 
-* [Create vector embeddings for your data](/docs/vectorizer-quick-start.md).
+**Working with embeddings generated from your data:**
+* [Automatically create and sync vector embeddings for your data](#automatically-create-and-sync-llm-embeddings-for-your-data)
+* [Search your data using vector and semantic search](#search-your-data-using-vector-and-semantic-search)
+* [Implement Retrieval Augmented Generation inside a single SQL statement](#implement-retrieval-augmented-generation-inside-a-single-sql-statement) 
```

Comment:
I would maybe not make the whole sentence a link, the blue is a little hard to read sometimes. Also so many links early in the Readme can be a bit overwhelming.